app/src/main/java/com/nextcloud/talk/chat/ChatActivity.kt

```diff
@@ -1429,30 +1427,17 @@ class ChatActivity :
     private fun initSmileyKeyboardToggler() {
```

Comment:
the keyboard toggler is somehow broken with the new solution. It's now possible to show the emoji picker and the keyboard at the same timebelow it, which is a bit confusing.